### DISCUSSION TITLE
How to setup etcd cluster on docker?

### DISCUSSION THREAD

**Discussion by FinalDes** [BretFisher/ama#169] [Q&A]
How to setup etcd cluster on docker, when I look at documentation, I don't understand some parameter how to configure?

**Comment by BretFisher** [comment]
I rarely setup etcd manually anymore, so I wouldn't know how to manually configure it. kubeadm and other Kubernetes tools do it for you if the purpose is for Kubernetes config storage.

**Comment by FinalDes** [comment]
I see some tools require setup etcd, whole you use separate etcd, or use the one in kubernetes cluster? @BretFisher

**Comment by BretFisher** [comment]
I would leave Kubernetes etcd just for it
